README.md

```diff
@@ -281,7 +281,7 @@ In an attempt to keep compatibility between different server and client versions
      "serverVersion": "1.0.0",
      "homeId": 3967882672,
      "minSchemaVersion": 0,
-     "maxSchemaVersion": 1
+     "maxSchemaVersion": 2
```

Comment:
I don't think that we need to update this. This is just an example connection string.

I do think that we should add a new `API_SCHEMA.md` with the changes introduced in each schema version.

Reply:
yes agreed. I started writing this up but I don't know what to put in for the previous schema changes. I can get a doc started and then someone can submit a PR to backfill info for schema 1